test/erllambda_ct.erl

```diff
@@ -56,6 +56,9 @@ record_to_map(R) when is_record(R, aws_config) ->
     record_to_map(R, Ks);
 record_to_map(R) when is_record(R, aws_assume_role) ->
     Ks = record_info(fields, aws_assume_role),
+    record_to_map(R, Ks);
+record_to_map(R) when is_record(R, hackney_client_options) ->
+    Ks = record_info(fields, hackney_client_options),
```

Comment:
why is this here?

Reply:
tests failed on my machine without such clause...

Reply:
well let's find out why and fix. this should not be here.